src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -22,7 +24,7 @@ public UserController(UserService userService, UserMapper userMapper) {
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto getByEmail(@RequestParam String email) {
+    public UserResponseDto getByEmail(@RequestParam @NotNull @Email String email) {
```

Comment:
These newest annotations should be only above the field in the specific Dto class.